recipe/meta.yaml

```diff
@@ -1,29 +1,42 @@
-{% set version = "1.8.0" %}
-{% set sha256 = "947a314a212b5d94985d89b43440dbe66b696e12bbdf9a2f78967b98d74abedc" %}
+{% set version = "1.14.1" %}
+{% set sha256 = "28d82b0d261544a9bf85b429399929e4986eb00efcf1ce16cc71d269a4c3186c" %}
 
 package:
   name: gstreamer
   version: {{ version }}
 
 source:
-  fn: gstreamer-{{ version }}.tar.xz
   url: https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-{{ version }}.tar.xz
   sha256: {{ sha256 }}
 
 build:
-  skip: true  # [win or osx]
-  number: 2
+  skip: true  # [win]
+  number: 0
+  script: install_gstreamer.sh
```

Comment:
Please remove this line. It seems the recipe in Anaconda is using a meta-recipe to build Gstreamer and its plugins, but that's not something we want here.